src/theme/components/globals.ts

```diff
@@ -0,0 +1,11 @@
+const globals = {
```

Comment:
Make this the same structure as we discussed

Reply:
fixed